build-system/dep-check-config.js

```diff
@@ -46,6 +46,10 @@ exports.rules = [
     filesMatching: '**/*.js',
     mustNotDependOn: 'third_party/**/*.js',
     whitelist: [
+      // TODO(levitzky) remove next line when experiment
+      // 'a4a-adsense-use-new-format' is removed.
+      'tools/experiments/experiments.js->' +
+          'third_party/babel/custom-babel-helpers.js',
```

Comment:
i don't quite understand why this is needed. i saw the dependency exists already